spec/lib/remote_resource/request_spec.rb

```diff
@@ -398,6 +398,26 @@ class RequestDummyWithCollectionPrefix < RequestDummy
       it 'returns the URL-encoded params' do
         expect(CGI.unescape(request.query)).to eql expected_query
       end
+
+      context "when connection_options[:use_body] is present" do
+        let(:connection_options) do
+          { root_element: :data, use_body: true, params: { pseudonym: 'pseudonym', labels: [1, '2', 'three'], pagination: { page: 5, limit: 15, ordered: true } } }
+        end
+
+        it 'returns nil' do
+          expect(request.query).to be_nil
+        end
+      end
+
+      context "when connection_options[:params][:use_body] is present" do
+        let(:connection_options) do
+          { root_element: :data, params: { use_body: true, pseudonym: 'pseudonym', labels: [1, '2', 'three'], pagination: { page: 5, limit: 15, ordered: true } } }
```

Comment:
I think I don't like supporting `use_body` as a param tbh; we should settle on a single approach

Reply:
or is there a reason for wanting to support both?

Reply:
That's just to cover both cases in specs; code in lib folder wasn't modified

Reply:
got it, but to add it to the gem, we should default to a single approach, I suggest removing the option to have use_body as a param.

Reply:
`use_body` is a bit unclear as well. Maybe something like: `params_from_body`, `use_params_in_body`, `read_body_for_params` or `body_params`?

Reply:
@p8 we can do that, but use_body is already in many places in SSP and CM, we will need to go though all those places too then

Reply:
@volosh1n-EP , that is the trade off of merging this to something reusable.

Perhaps `force_send_params_in_body`? (it is an instruction that won't / shouldn't reach the server)